lib/rules/disallow-function-prototype-extension.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+var includes = require("../utils/includes");
+var extensionFixes = {
+  'property': "Use the computed('dependentKey', function() {}) syntax instead.",
+  'observes': "Use the observer('dependentKey', function() {}) syntax instead.",
+  'observesBefore': "Also, using before observers is deprecated."
```

Comment:
Should include `on` and `observesImmediately`:

![](https://api.monosnap.com/rpc/file/download?id=sOenv3OFFWGAe8oUKIZbQtCHVboWp7)